src/MentionsTextInput.js

```diff
@@ -611,10 +612,11 @@ export default class MentionsTextInput extends Component {
   }
 
   onContentSizeChange(event) {
+    const extraHeight = this.props.extraContentSizeHeightOffset || 0;
```

Comment:
Since zero is a legitimate value for `this.props.extraContentSizeHeightOffset` I would use the ?? operator instead of || (although it won't change the evaluation of this expression.